app/views/documents/show/_featured_attachments.html.erb

```diff
@@ -7,5 +7,15 @@
       { href: featured_attachments_path(@edition.document),
         data_attributes: { gtm: "edit-attachments" } }
     end
+  ),
+  items: (
+    @edition.featured_attachments.each_with_index.map do |attachment, index|
+      {
+        field: index + 1,
+        value: sanitize(
+          "<h4 class='govuk-heading-s govuk-!-margin-bottom-0'>#{attachment.title}</h4>#{render('components/attachment_meta', { attachment: file_attachment_attributes(attachment, @edition) })}"
```

Comment:
Nice one for working out how to get this in - I think I would have struggled to find the right combination! I wonder if this suggests a smell about the `summary_list` component: we're using `field` to mean a number here; perhaps a cleaner approach (involving less manipulation here), would be for the component to support a "numbered" attribute.

What do you think?

Reply:
We do already [use a number as a key in collections-publisher](https://github.com/alphagov/collections-publisher/blob/896b0a145e25d4f68926b39724656290ac4102ac/app/views/step_by_step_pages/show.html.erb#L30), although sometimes it's a word like "AND" or "OR", so not quite as simple.

Are you proposing this sort of structure?

```erb
{
  numbered: true,
  field: "<h4 class='govuk-heading-s govuk-!-margin-bottom-0'>#{attachment.title}</h4>",
  value: render('components/attachment_meta', { attachment: file_attachment_attributes(attachment, @edition) })}
}
```

I wonder how this should be presented in the underlying [description list/terms/descriptions](https://www.w3schools.com/tags/tag_dd.asp) markup. For reference, this is our current output:

```html
<dl>
  <dt>1</dt>
  <dd><h4>Foo</h4>Unique ref</dd>
</dl>
```

Looking over this, I think the implementation is probably incorrect. We're using the summary list because it's styled the way we want it, but semantically this should really be an ordered list instead.

```html
<ol>
  <li><h4>Foo</h4>Unique ref</li>
</ol>
```

I think a `numbered` attribute is probably fixing the wrong problem, and suggest we mark up our own `<ol>` to look like the summary list instead. (If/when we have time, of course...!)

Reply:
Yes, that's right. I also had a quick look at the component, and although I can't see any huge semantic issues with describing the items this way, there is at least room to improve the interface so we don't have to fake a numbered list ourselves. And perhaps the enhanced interface would be implemented as an ordered list, like you suggest.

I think there is one thing we could do here: would you be willing to run this by [the design system team](https://github.com/alphagov/govuk-design-system-backlog/issues/182)? Perhaps we can incorporate this need into the design system.

Reply:
Now done: https://github.com/alphagov/govuk-design-system-backlog/issues/182#issuecomment-610387202